### PR TITLE
another sync of changes from my other repo

### LINKED ISSUE #118: Enable optional advancements?
Mainline's [optional_unit_advancements.cfg](https://github.com/wesnoth/wesnoth/blob/master/data/core/macros/optional_unit_advancements.cfg) contains a bunch of optional advancements that we could enable: https://www.wesnoth.org/macro-reference.html#file:optional_unit_advancements.cfg

Friendly ones:
- `{ENABLE_PARAGON}` (so Majid Yahyazad can finally turn into @knyghtmare)
- `{ENABLE_WOSE_SHAMAN}` (only relevant if/when #117 gets merged, and even then I doubt any allied woses would live that long...)

Hostile ones:
- `{ENABLE_ARMAGEDDON_DRAKE}` (only relevant if/when #117 gets merged, and even then I doubt any enemy drakes would live that long...)
- `{ENABLE_NIGHTBLADE}`
- `{ENABLE_TROLL_SHAMAN}`
- `{ENABLE_ANCIENT_LICH}`
- `{ENABLE_DEATH_KNIGHT}`

So, what do people think? Does it make sense to enable any of these? Perhaps only on certain difficulty levels?

### PR DESCRIPTION
**Pull request by cooljeanius**
- create CODEOWNERS file, to ensure that the primary people who have edited files get notified when changes to them are proposed (I'm the only one in it so far, as anyone else who it'd make sense to add would have to be added as a collaborator to this repo first)
- enable optional advancements; closes #118
- confirmation of @alxrem's translation credits entry
- start on ChangeLog entries for next release (which will be 1.7.0)
- updated Medic frankenportrait from @DuncanusDillius
- even more validator fixes that I had thought were too difficult to do previously
- start writing support conversations (in [macros/conversations_support.cfg](https://github.com/nemaara/A_New_Order/compare/master...cooljeanius:A_New_Order-1:start_writing_support_convos?expand=1#diff-69f97f46055f5cc107057d99f22a48cef628a6b48e0a03608190e433a52f44a6)), for issue #73 
- improve S01 cutscene/dialogue
- attempt at debugging odd bug where Lorin randomly disappears when visiting the gallows in S02
- minor updates to the Akladian name generator

### CODE REVIEW

**Comment by cooljeanius** [comment]
@nemaara any thoughts?

**Comment by nemaara** [comment]
it's 1000 lines so lemme comment tmrw or sth

**Comment by cooljeanius** [comment]
> it's 1000 lines so lemme comment tmrw or sth

all of a sudden I've gotten all self-conscious about the support convos... feel free to tell me to leave those out if they make things too difficult to review...

**Comment by cooljeanius** [comment]
> > it's 1000 lines so lemme comment tmrw or sth
> 
> all of a sudden I've gotten all self-conscious about the support convos... feel free to tell me to leave those out if they make things too difficult to review...

...ok, I trimmed them down a bit by removing one of them...

**Comment by cooljeanius** [comment]
ok I got it down under 1000 lines with [481329d](https://github.com/nemaara/A_New_Order/pull/171/commits/481329d70071d49264f122ff9e0d9ef4eef938e8) (most of the lines are in the potfile)

**Comment by nemaara** [comment]
I think some of the text could use revising but I don't have time to do that right now. I could add it in as is or would you like to try a second time at rewriting some lines? Basically I'm not sure some of the characters' personalities were captured in the newer text.

**Comment by cooljeanius** [comment]
> I think some of the text could use revising but I don't have time to do that right now. I could add it in as is or would you like to try a second time at rewriting some lines? Basically I'm not sure some of the characters' personalities were captured in the newer text.

I mean, I guess another thing I could do would be to split the support convos off into a separate PR? Does everything else besides the support convos look fine?

File: scenarios/01_Breaking_the_Circle.cfg [hunk 1]
```diff
@@ -672,19 +674,66 @@
...
+                [/heal_unit]
                 {CLEAR_VARIABLE ano_gawen_poisoned}
+                [delay]
+                    time=234
+                [/delay]
+                [redraw][/redraw]
+                [delay]
+                    time=234
+                [/delay]
+                {MESSAGE (Medic) () (_"Medic") _"...and there you go."}
```
Comment (nemaara) [inline scenarios/01_Breaking_the_Circle.cfg:732]: Probably doesn't need a message.

File: scenarios/01_Breaking_the_Circle.cfg [hunk 2]
```diff
@@ -672,19 +674,66 @@
...
+                {MSG_Gawen _"Thanks. (*<i>faints</i>*)"}
```
Comment (nemaara) [inline scenarios/01_Breaking_the_Circle.cfg:733]:
```suggestion
                {MSG_Gawen _"Thank you. (*<i>faints</i>*)"}
```

**Review by nemaara — APPROVED** [review]
Besides support convos the rest is fine.

File: scenarios/01_Breaking_the_Circle.cfg (hunk 1 above)
Comment (cooljeanius) [reply scenarios/01_Breaking_the_Circle.cfg:732]: I think it helps to mark the end of the animation, and to give another chance to see the portrait

Comment (cooljeanius) [reply scenarios/01_Breaking_the_Circle.cfg:732]: ...I'll add a comment to that effect.

Comment (cooljeanius) [reply scenarios/01_Breaking_the_Circle.cfg:734]: Comment added in [8029e28](https://github.com/nemaara/A_New_Order/pull/171/commits/8029e2809a6f79ad9dee5da1030af349783d9dc6).

**Pull request merged**